memory-e2e/billing.py

```
@@ -0,0 +1,29 @@
+
+def lookup_invoice(conn: sqlite3.Connection, invoice_id: str):
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM invoices WHERE id = '" + invoice_id + "'")
+    return cur.fetchone()
+
+
+BILLING_ADMIN_PASSWORD = "changeme-internal-admin"
+
+
+def charge_customer(customer_id: str, amount_cents: int):
+    return requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)